### ISSUE TITLE
Neutrino Generator puts MMCTrack list in Frame, which causes a clash with the MMC module in some cases. (Trac #325)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#324]
Neutrino Generator puts an MMCTrack list into the frame that causes unwanted clashes with MMC. It does this even when it is explicitly told not to

Migrated from https://code.icecube.wisc.edu/ticket/325
```json
{
    "status": "closed", 
    "changetime": "2014-11-22T18:26:26", 
    "description": "Neutrino Generator puts an MMCTrack list into the frame that causes unwanted clashes with MMC. It does this even when it is explicitly told not to", 
    "reporter": "icecube", 
    "cc": "Peter, Redl", 
    "resolution": "fixed", 
    "_ts": "1416680786794380", 
    "component": "combo simulation", 
    "summary": "Neutrino Generator puts MMCTrack list in Frame, which causes a clash with the MMC module in some cases.", 
    "priority": "normal", 
    "keywords": "V02-06-03", 
    "time": "2011-11-04T18:16:48", 
    "milestone": "", 
    "owner": "Alex Olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20111104T18:23:42: olivas changed milestone from "" to "IceSim 3.1 Release"

**Comment by icecube-trac** [comment]
Trac update at 20120229T05:15:47: 
* olivas changed resolution from "" to "fixed"
* olivas commented:

> Fixed in IceSim 2.6.4.


* olivas changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20141122T18:26:26: 
* olivas changed cc from "Peter Redl" to "Peter, Redl"
* olivas commented:

> Milestone IceSim 3.1 Release deleted


* olivas changed milestone from "IceSim 3.1 Release" to ""
